tgm-example-plugin.php

```diff
@@ -1,43 +1,68 @@
 <?php
-/*
-Plugin Name: TGM Example Plugin
-Plugin URI: https://github.com/thomasgriffin/TGM-Plugin-Activation
-Description: This is an example plugin to going along with the TGM Plugin Activation class.
-Author: Thomas Griffin
-Author URI: http://thomasgriffinmedia.com/
-Version: 1.0.0
-License: GNU General Public License v3.0
-License URI: http://www.opensource.org/licenses/gpl-license.php
-*/
-
-/*  Copyright 2011  Thomas Griffin  (email : thomas@thomasgriffinmedia.com)
-
-    This program is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License, version 3, as 
-    published by the Free Software Foundation.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License
-    along with this program; if not, write to the Free Software
-    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-    
-*/
-
-
-add_action( 'rightnow_end', 'tgm_php_mysql_versions', 9 );
 /**
- * tgm_php_mysql_versions function.
- * 
- * This function displays the current server's PHP and MySQL versions right below the WordPress version in the Right Now dashboard widget.
+ * TGM Example Plugin.
+ *
+ * Example plugin to demonstrate the ability to handle bundled plugins with the
+ * TGM Plugin Activation library.
  *
- * @since 1.0.0
+ * @package     WordPress\Plugins\TGM Example Plugin
+ * @author      Thomas Griffin <thomas@thomasgriffinmedia.com>
+ * @link        https://github.com/thomasgriffin/TGM-Plugin-Activation
+ * @version     1.0.1
+ * @copyright   2011-2016 Thomas Griffin
+ * @license     http://creativecommons.org/licenses/GPL/3.0/ GNU General Public License, version 3 or higher
  *
+ * Plugin Name: TGM Example Plugin
```

Comment:
The `@wordpress-plugin` tag is needed before this line. Non-tagged comments are not valid after PHPDoc tags.